### PR TITLE
Sideloader this thing.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
Not ready for review yet, just getting a look at the diff.

**Comment by smn** [comment]
@LyndsayLawrence Will deploy tomorrow morning and we'll see where things go from there.

File: grails-app/conf/BuildConfig.groovy [hunk 1]
```diff
@@ -50,7 +50,7 @@ grails.project.dependency.resolution = {
     dependencies {
         // specify dependencies here under either 'build', 'compile', 'runtime', 'test' or 'provided' scopes e.g.
         // runtime 'mysql:mysql-connector-java:5.1.29'
-        // runtime 'org.postgresql:postgresql:9.3-1101-jdbc41'
+        runtime 'org.postgresql:postgresql:9.3-1101-jdbc41'
```
Comment (smn) [inline grails-app/conf/BuildConfig.groovy:53]: @LyndsayLawrence can you explain if this is needed or if I'm doing something weird to get this thing to run?

File: grails-app/conf/DataSource.groovy [hunk 2]
```diff
@@ -31,7 +31,8 @@ environments {
     production {
         dataSource {
             dbCreate = "update" // one of 'create', 'create-drop','update'
-            url = "jdbc:postgresql://localhost:5432/scheduler"
+            uri = new URI(System.env.DATABASE_URL?:"postgresql://localhost:5432/scheduler")
+            url = "jdbc:postgresql://" + uri.host + ":" + uri.port + uri.path
```
Comment (smn) [inline grails-app/conf/DataSource.groovy:35]: @LyndsayLawrence please advise if there's a better way of doing this.

File: scripts/postinstall.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+docker rm mama-ng-scheduler
+
+docker stop postgresql
+docker rm postgresql
+
+# Exit on errors from here.
+set -e
+
+docker run -d --name="postgresql" \
+    -v `pwd`/data:/data:rw \
```
Comment (smn) [inline scripts/postinstall.sh:13]: mounts a local volume for the data, postgres will write its DBs here ensuring it lives on during redeploys.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**